### ISSUE TITLE
Of numbers & vegetation plots

### ISSUE THREAD

**Issue by lenjon** [fmsabatini/sPlotOpen_Manuscript#161]
I am very confused with the different numbers of vegetation plots reported in the current ms. At some point in the ms, you wrote that there is 107,238 vegetation plots across the three resampling iterations. These are unique plots right? I mean with unique IDs (no duplicates), right? How exactly do you end up with this number? Indeed, if I sum the number of plots across all three resampling iterations, it is 49,787 + 49,811 + 49,789 = 149,387, and if I substract the overlapping plots (19,672 plots overlapping across the three iterations and 14,939 plots overlapping across two iterations), this makes a total of 114,776 plots. So why do I get a different number from the one you report in the section entitled "Permission to release the data as open access"? Sorry @fmsabatini but I am rather confused and I guess the reader will too, no? Maybe worth cross-checking all those numbers. 

@fmsabatini I think that I got it now about the numbers. I was just missing one step describing the initial number of plots per resampling iteration before removing the plots not granted as open access. I have made my edits and changes accordingly in the ms. I will soon share with you my PR with my proposed changes throughout the ms.

**Comment by lenjon** [comment]
Another issue on line 40 of the "Background & Summary" section is the total number of unique vegetation plots reported which is 94,210 and thus different from the numbers reported in the abstract or in the caption of figure 1 (95,104). Which number is the correct one here: 94,210 or 95,104?

https://github.com/fmsabatini/sPlotOpen_Manuscript/blame/10031bb032ff195182e878854ca97f6e82321a4e/content/20.BackgroundSummary.md#L40

**Comment by fmsabatini** [comment]
https://github.com/fmsabatini/sPlotOpen_Manuscript/blame/master/content/20.BackgroundSummary.md#L40
this was a mistake and is now fixed

**Comment by lenjon** [comment]
@fmsabatini Ok, thanks for correcting this mistake. But how then do you get the total of 95,104 vegetation plots? I mean, if I do the difference 107,238 minus 12,035 (not granted as open access) I get a total of 95,203. The difference is small but still... Where does that come from? About the resampling iteration #1, does figure 2 show the set of plots without or with the reserve? I assume it is without the reserve, right? Maybe specify clearly in the caption of figure 2 the size of the resampling datatset.

**Comment by fmsabatini** [comment]
I solved the mystery. 
107,238 - unique plots
95,104 - unique, usable plots
11069  - Plots that we cannot use
966 - Plots for which we never receive authorization (i.e., we cannot use).

The remaining 99 plots are plots which were selected in the algorithm, but belong to a dataset which was withdrawn from sPlot about 18 months ago, and therefore deleted from our 'header' file.
As a matter of fact, these are plots that we cannot use, so we'd better add them to that number
12035+99=12134

**Comment by lenjon** [comment]
@fmsabatini Ok, I see. Then, I think that we should at least adjust the number 12035 to 12134 so that every numbers are adjusting correctly and thus somepne doing the maths will see that it fits well. Either we provide the full explanation like you did here or we do not provide these details about the 12134 plots that we cannot use but we at least change the number 12035 to 12134 so taht everything is matching perfectly. Personnaly, I would be transparent and just report that in the ms like you did here. Then, we will have all explanations clear :)

**Comment by fmsabatini** [comment]
Thanks a lot @lenjon for double-checking the numbers and clarifying the text. It's much better now. 
I made a final read (with fresh brain, of the sections 40 and 50). I built on top of your edits and tried to clarify the whole story even further. 
I kept referring to the 'consensus' version in the ms, which I think is now much better explained. I don't have a strong opinion, however, on whether we should keep it, or delete it. I suggest to wait for the referee comments.
